src/lib/observe.ts

```diff
@@ -30,19 +30,21 @@ interface PerformanceEntriesHandler {
  * try/catch to avoid errors in unsupporting browsers.
  */
 export const observe = (
-  type: string,
+  types: string[],
```

Comment:
I considered making this `string || string[]` and adjusting this implementation to support either option... decided it was better to make a single choice but totally understand alterantive.